src/main/resources/lang/default.lang

```diff
@@ -1973,7 +1973,7 @@ types:
 	inventory: inventor¦y¦ies @an
 	player: player¦s @a
 	offlineplayer: offline player¦s @a
-	commandsender: player¦¦s¦/console @a
+	commandsender: command sender¦s @a
```

Comment:
I find this makes the error more confusing, honestly. As a user I am confused why Skript is talking about command senders in a click event. The previous one wasn't much better, but I think this is a step sideways, not forwards. Sadly, I don't really have any better suggestions.
![image](https://github.com/SkriptLang/Skript/assets/10354869/86a8e741-1cf4-4c98-8b0c-f95643cc4636)

Reply:
Clicking on the command sender type in the documentation takes you to the description of the type which is better way to explain what the type represents.